### DISCUSSION TITLE
Where is the logic for running the app?

### DISCUSSION THREAD

**Discussion by quackduck** [hackclub/hack-as-a-service-v0#135] [Q&A] [answered]
I can't seem to find it.

**Comment by cjdenio** [comment, marked as answer]
The API's entrypoint is `main.go`, and the rest of the code is located in `pkg/`.

If you're looking for a dev environment, use `docker-compose up -d`

**Reply by quackduck** [reply]
Alright, but where is the function call to say, run the command to deploy the app?

**Reply by gleich** [reply]
@quackduck check out this file: https://github.com/hackclub/hack-as-a-service/blob/master/pkg/api/apps/deploy.go
